src/components/app/Details.css

```diff
@@ -20,3 +20,14 @@
 .sidebar-open-close-button-isclosed {
   background-image: url(../../../res/img/svg/pane-expand.svg);
 }
+
+.tabBarContainer {
```

Comment:
If you intend to keep this here, it would probably be better to have a more specific classname; (but also see the general comment)

Reply:
Renamed to `.Detais-top-bar`.